src/objects/embed/embed.scss

```diff
@@ -0,0 +1,119 @@
+@use 'sass:meta';
+@use '../../design-tokens/aspect-ratio.yml';
+@use '../../design-tokens/breakpoint.yml';
+
+/**
+ * We define the default ratio up-front to avoid outputting unnecessary modifier
+ * classes for it.
+ */
+
+$default-aspect-ratio: aspect-ratio.$square;
+
+/* stylelint-disable suitcss/custom-property-no-outside-root */
+
+/**
+ * Makes ratio modifier classes based on the `aspect-ratio` and `breakpoint`
+ * design tokens. Intended to be included from within the base class.
+
+ * We iterate over breakpoints before ratios so wider breakpoints will be later
+ * in the cascade. If we iterated over aspect ratios first, then a narrower
+ * breakpoint would override a wider one if its aspect ratio came first in our
+ * design token file.
+ *
+ * 1. For some reason Sass does not compile `$ratio` without this string
+ *    interpolation syntax. I'm not entirely sure why.
```

Comment:
I ran into this the other day. This might be a helpful link: https://sass-lang.com/documentation/breaking-changes/css-vars

Reply:
That is a helpful link!